hrms/hr/doctype/interview/interview.py

```diff
@@ -217,8 +217,9 @@ def send_daily_feedback_reminder():
 	interview_feedback_template = frappe.get_doc(
 		"Email Template", reminder_settings.feedback_reminder_notification_template
 	)
+
 	interviews = frappe.get_all(
-		"Interview", filters={"status": ["in", ["Under Review", "Pending"]], "docstatus": ["!=", 2]}
+		"Interview", filters={"status": ["in", ["Under Review", "Pending"]], "docstatus": ["!=", 2], "scheduled_on": [ ">=", getdate(today())], "to_time": [ ">", get_time(now())]}
```

Comment:
```suggestion
		"Interview", filters={"status": "Under Review"], "docstatus": ["!=", 2], "scheduled_on": [ ">=", getdate(today())], "to_time": [ ">", get_time(now())]}
```
Interview Feedback is to be sent after the interview is over. This will only send feedback reminders for interviews scheduled in the future
`"scheduled_on": [ ">=", getdate(today())]`

Instead, reminders could be sent based on the status, maybe just "Under Review"
Feedback marked as "Under Review" is applicable for feedback

Reply:
Yes,
scheduled_on condition will be "scheduled_on": [ ">=", getdate(today())]

But we may need to consider the Interview end time also, Becuse if we didn't consider end time it will send to future interviews also.

Let's consider following cases and assume current date is 18/07/2023 and time is 12:00:00 PM

Case 1 :
Interview date : 18/07/2023
Interview Time : 10:00:00 AM to 12:00:00 PM
Expected : Reminder need to send

Case 2 :
Interview date : 18/07/2023
Interview Time : 09:00:00 AM to 11:00:00 PM
Expected : Reminder need to send

Case 3 :
Interview date : 18/07/2023
Interview Time : 10:00:00 AM to 12:30:00 PM
Expected : Reminder not need to send

Case 4 :
Interview date : 18/07/2023
Interview Time : 13:00:00 AM to 15:30:00 PM
Expected : Reminder not need to send